en/sql/authorization.rst

```diff
@@ -114,6 +114,7 @@ You don't need to grant authorization individually because the members of the gr
     *   **EXECUTE**: Allows to call table methods or instance methods.
     *   **ALL PRIVILEGES**: Includes all permissions described above.
 
+* *schema_name*: Specifies the schema name of the table or virtual table. If omitted, the schema name of current session is used.
```

Comment:
```suggestion
* *schema_name*: Specifies the schema name of the table or virtual table. If omitted, the schema name of the current session is used.
```

Reply:
Thank you. I will change other similar sentences.